bin/sirius-script-rabpm-mngr.py

```diff
@@ -43,6 +43,17 @@ def run_rffe_reset(rack, virtual_slot):
     execute_ssh(hostname, "rffe-reset", virtual_slot)
 
 
+def run_boot_from_flash(rack, slot):
+    hostname = get_hostname(rack)
+    board = ""
+    if (slot == 2):
+        board = "afcv4-sfp"
+    else:
+        board = "afcv3"
```

Comment:
No need for parentheses, and `board` doesn't have to be declared before use.

But since you wanted a ternary, you can use `board = "afcv4-sfp" if slot == 2 else "afcv3"`